npc/quests/quests_13_1.txt

```diff
@@ -2922,7 +2922,7 @@ mid_campin,106,122,4	script	Receptionist Brink#ep13b	89,{
 	}
 }
 
-mid_camp,264,263,4	script	Diego#ep13bs	931,{
+mid_camp,266,263,4	script	Diego#ep13bs	931,{
```

Comment:
```
//npc "mid_camp" "Diego#ep13bs" 4_M_RACHMAN1 264 263 4 0 0
npc "mid_camp" "Diego#ep13bs" 4_M_RACHMAN1 266 263 4 0 0
```
266 for renewal. Split the npc into re/pre-re

Reply:
Right, thanks!